pkg/networkservice/common/heal/eventloop.go

```diff
@@ -32,18 +31,19 @@ import (
 	"github.com/networkservicemesh/sdk/pkg/tools/log"
 )
 
-const attemptAfter = time.Millisecond * 200
+const maxAttemptsWithoutReselect = 3
```

Comment:
@xzfc Could you say more why do we need this?

Reply:
I've added it as a workaround to [this issue](https://github.com/networkservicemesh/sdk/issues/1187#issuecomment-1002990951) (NSE pod was deleted, but it still was responding to pings), not knowing that this was fixed in https://github.com/networkservicemesh/sdk-vpp/pull/537.

However, I think it still makes sense to perform a forceful reselect after a few failed requests to the same endpoint, even if the data plane is still alive.

Reply:
We should keep requests without reselect while datapath is fine.

Reply:
Removed `maxAttemptsWithoutReselect`.